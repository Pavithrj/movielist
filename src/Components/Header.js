import React from 'react';
import './Header.css';

function Header(props) {
    console.log(props);
    const { title } = props;
    return (
        <div className='headerStyle'>
            <h1 id='title'>{title}</h1>
        </div>
    );
}

export default Header;