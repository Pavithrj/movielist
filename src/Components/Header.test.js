import { render } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Header from './Header';


describe("Header Component", () => {
    it('renders learn react link', () => {
        const { asFragment } = render(<Header title="test title"/>);
        expect(asFragment()).toMatchSnapshot();
    });
})