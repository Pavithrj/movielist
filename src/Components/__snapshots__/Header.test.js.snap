// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="headerStyle"
  >
    <h1
      id="title"
    >
      test title
    </h1>
  </div>
</DocumentFragment>
`;
